assets/css/style.css

```diff
@@ -0,0 +1,663 @@
+  font-weight: 500;
+  color: rgb(127, 140, 255);
+  margin-top: 14px;
+  margin-bottom: 14px;
+}
+
+.heading__icons {
+  margin: 0.5rem 0;
+}
+
+.heading__icons__ul {
+  align-items: center;
+  justify-content: space-between;
+  width: 11rem;
+}
+
+.tonic {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  background-color: #fff;
+  border-radius: 24px;
+  border: solid 1px rgb(223, 225, 230);
+  align-content: flex-start;
+}
+
+.tonic:hover {
+  box-shadow: 0 48px 48px rgba(37.12, 47.22, 137.06, 0.08);
+  border: solid 1px rgb(96, 112, 255);
+}
+
+.content .title {
+  font-weight: 700;
+  color: #091e42;
+  margin-bottom: 12px;
+}
+
+.content .desc {
+  color: rgb(52, 69, 99);
+  margin-bottom: 24px;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.content .summary {
+  margin-bottom: 24px;
+}
+
+.content .summary span {
+  margin-right: 12px;
+}
+
+.content .summary .dotted {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  background-color: rgb(193, 199, 208);
+  border-radius: 9999px;
+}
+
+.content .summary .color {
+  color: rgb(107, 119, 140);
+}
+
+.content .tagging {
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  margin-bottom: 40px;
+}
+
+.tagging li {
+  padding-right: 5px;
+}
+
+.about {
+  background-color: #fff;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin-bottom: -50px;
+}
+
+.about .casualose .border-bottom {
+  border-bottom: 1px solid #d9dbdf;
+}
+
+.about .casualose .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 10px 0;
+}
+
+.about .casualose .header span {
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  color: rgb(52, 69, 99);
+}
+
+.about .casualose .language li {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  border-radius: 8px;
+  margin: 10px 0;
+  padding: 12px 8px;
+  background: #f7f7f9;
+}
+
+.footer {
+  flex-direction: column;
+  position: relative;
+  border-top-left-radius: 5rem;
+  z-index: 10;
+  background-color: rgb(96, 112, 255);
+  justify-content: center;
+  align-items: center;
+}
+
+.footer .decoration {
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.footer .title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: white;
+  margin-bottom: 12px;
+}
+
+.footer .desc {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: rgb(235, 235, 255);
+  margin-bottom: 20px;
+}
+
+input,
+textarea {
+  padding: 15px;
+  margin: 9px 0;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(207, 216, 220);
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: rgb(23, 43, 77);
+}
+
+::placeholder {
+  color: rgb(179, 186, 197);
+}
+
+.footer .contact-button {
+  padding-top: 12px;
+}
+
+.about .info .button-outline {
+  margin-top: 0.5rem;
+  margin-bottom: 1.5rem;
+}
+
+@media all and (min-width: 375px) {
```

Comment:
- **[OPTIONAL]** There are some differences between your design and the Figma design. As requested by the previous CR it would have been enough to mark one breakpoint at width 768px for the desktop version 👉🏽 **@media screen and (min-width: 750px)** It will save you time to have two different styles one for the mobile version and the other for versions higher than 768px width.